.github/workflows/test.yml

```diff
@@ -0,0 +1,98 @@
+name: Example Github Workflow
+on:
+  push:
+  workflow_dispatch:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight
+
+# env:
+  # CLI_INSTALLER_SKIP_CACHE: true
+
+jobs:
+  install:
+    name: Installation Example
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "macos-latest", "windows-latest", "ubuntu-20.04" ]
+        cache: [ true, false ]
+        # os: [ "ubuntu-20.04" ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./
+        id: install_clients
+        with:
+          # crc: "*"
+          source: "github"
+          github_pat: ${{ github.token }}
+          s2i: "latest"
+          kam: "latest"
+          kamel: "latest"
+          opm: "latest"
+          tkn: "latest"
+          # "operator-sdk": "1.6.1"         # Not available on windows
+          # helm: "3.5"
+          # kam: "1"
+          # kamel: "1"
+          # kn: "0.15"
+          # oc: "4"
+          # odo: "2"
+          # # operator-sdk: "4.7"         # Not available on windows
+          # opm: "4.6"
+          # # openshift-install: "4.x"    # Not available on windows
+          # # opm: "4"                    # Only available on linux
+        env:
+          CLI_INSTALLER_SKIP_CACHE: ${{ matrix.cache }}
```

Comment:
I realize, now that we have inputs that aren't executable names, we should move the 'skip cache' toggle to an input too rather than env.

Reply:
Yeah makes sense.